packages/amesos2/example/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 
-# TRIBITS_ADD_EXECUTABLE_AND_TEST(
-#   SimpleSolveExample
-#   SOURCES SimpleSolve
-#   ARGS
-#   COMM serial mpi
-# )
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  SimpleSolveExampleNonContigMap
+  SOURCES SimpleSolveNonContigMap
+  ARGS
```

Comment:
Let us not replace the SimpleSolveExample, but add a new one SimpleSolveExampleNonContigMap in addition to the existing one.